content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+     --network prometheus \
+     quay.io/hpestorage/array-exporter:v1.0.0-beta \
+     --log.path /var/log/hpe-array-exporter.log \
+     --accept-eula \
+     /etc/config/storage-system.yaml
+```
+
+By running the above command with the `--accept-eula` parameter, the end-user accepts the HPE End User License Agreement at <https://www.hpe.com/us/en/software/licensing.html>.
+
+The exporter is now up and running listening on port 8080. To ensure we can reach the metrics endpoint, run the following `docker` command:
+
+```markdown
+docker run --rm --network prometheus \
+    curlimages/curl -s http://my-array-1:8080/metrics
+```
+
+It should produce an output that enumerates the metric keys, value, type and brief help. Example:
+
+```markdown
+# HELP hpenimble_pool_snapshot_used_bytes Total logical size of all snapshots in a storage pool
+# TYPE hpenimble_pool_snapshot_used_bytes gauge
+hpenimble_pool_snapshot_used_bytes{pool="default"} 2.659803136e+09
+```
+
+Next, let's start Prometheus and scrape some metrics.
+
+# Run and Inspect Prometheus
+
+When Prometheus is deployed on Kubernetes, discovery of scrape targets may be configured to be performed dynamically. In this example we need to instruct Prometheus where to find our target. Create a configuration file named `prometheus.yml`:
+
+```yaml
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "my-array-1"
+    static_configs:
+      - targets: ["my-array-1:8080"]
+```
+
+Create the Prometheus container:
+
+```markdown
+docker run -d --rm --name prometheus-server \
+    --network prometheus \
+    -p 80:9090 \
+    -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus:v2.30.3
+```
+
+In a few moments, the Prometheus server should be up and listening on the docker host and accessible from http://your.docker.host.example.com/. 
+
+To inspect if metrics are being collected properly, access the web interface and start typing "hpe" in the query bar.
+
+![Prometheus Query](/img/prometheus-auto.png)
+
+Select a \`volume_reads_per_second\` metric, hit "Execute" and click "Graph". This should produce a filled line graph with all the volumes on the array and their read IOPS. Wait a few minutes if there's no data being presented.
+
+![Prometheus Graph](/img/prometheus-graph.png)
+
+Next, we'll bolt on Grafana and connect to the Prometheus data source.
+
+# Run Grafana and Build Your First Dashboard
+
+In this tutorial we'll just use the built-in dashboards and plugins that come with Grafana. Accessing Grafana is mainly done through the intuitive web UI found on port 3000. Let's start it up:
+
+```markdown
+docker run --rm -d --name grafana \
+    --network prometheus \
+    -p 3000:3000 \
+    grafana/grafana:8.2.2
+```
+
+Next, navigate to the web UI. http://your.docker.host.example.com:3000/, login with user "admin" and password "admin", you'll be prompted to create a new password. Reusing "admin" for this example is perfectly fine.
```

Comment:
Recommend hotlinking the URL on "web UI"
Change "login" to "log in" (since you are using it as a verb).
I'd recommend breaking up the first sentence, modifying to:
Next, navigate to the web UI and log in with user "admin" and password "admin". You'll be prompted to create a new password.

Reply:
The URL is unique depending on how you run docker in your environment, you need to understand the nuance of where docker present applications. I made some adjustments here.